script/Arcueid/Day 3/03_00_ARC03_2B.txt

```diff
@@ -112,7 +112,7 @@ Kohaku-san gives me a wide grin.
 -- Page 92, Offset 4188.
 -- もともとむこうで使ってたのは有間の家のものだったから、自分の物なんて着てる服ぐらいで……」
 Everything I used over there originally belonged to the Arima family.
- Pretty much all I own is the clothes on my back..."
+ Pretty much all I own are the clothes on my back..."
```

Comment:
`is` is definitely like 'wrong' grammar but weirdly it feels correct to me? I also get more (9k to 3k) google strict hits for `is` over `are` with a phrase search.

Reply:
prefer latter also